app/assets/javascripts/3dbio_viewer/src/webapp/components/network/NetworkForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useState, useRef } from "react";
+import i18n from "../../utils/i18n";
+import { Dropzone, DropzoneRef } from "../dropzone/Dropzone";
+import IncludeNeighborsCheckbox from "./IncludeNeighborsCheckbox";
+import Label from "./Label";
+import NetworkExample from "./NetworkExample";
+import UniProtAccessionTextArea from "./UniProtAccessionTextArea";
+import SpeciesSelect from "./SpeciesSelect";
+import "./Network.css";
+
+interface NetworkForm {
+    species: string;
+    uniProtAccession: string;
+    includeNeighboursWithStructuralData: boolean;
+}
+
+function getInitialNetworkForm(): NetworkForm {
+    return {
+        species: "homoSapiens",
+        uniProtAccession: "",
+        includeNeighboursWithStructuralData: false,
+    };
+}
+
+const NetworkForm = React.memo(() => {
+    const annotationFileRef = useRef<DropzoneRef>(null);
+    const [error, setError] = useState<string>();
+    const [networkForm, setNetworkForm] = useState<NetworkForm>(() => getInitialNetworkForm());
+
+    const addNetwork = useCallback(() => {
+        setError("");
+        if (networkForm.uniProtAccession === "") {
+            setError("Error: Please write down the UniProt accession");
+        }
+    }, [networkForm]);
+
+    return (
+        <form className="network-form">
+            <Label forText="species" labelText="Select species" />
+            <SpeciesSelect
+                value={networkForm.species}
+                onSpecieChange={e =>
```

Comment:
`e` is too short, event? but's not the event, it's the species. Maybe `species` or `newSpecies`.